packages/react-components/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -26,26 +26,31 @@ const useRootStyles = makeStyles({
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
     ...shorthands.borderWidth(0),
     cursor: 'pointer',
-    display: 'flex',
-    flexDirection: 'row',
+    display: 'grid',
     flexShrink: 0,
+    gridAutoFlow: 'column',
+    gridTemplateColumns: 'auto',
+    gridTemplateRows: 'auto',
     fontFamily: tokens.fontFamilyBase,
     lineHeight: tokens.lineHeightBase300,
     outlineStyle: 'none',
     position: 'relative',
     ...shorthands.overflow('hidden'),
     textTransform: 'none',
   },
+  horizontal: {
+    justifyContent: 'center',
+  },
+  vertical: {
+    justifyContent: 'flex-start',
+  },
   mediumHorizontal: {
     columnGap: tokens.spacingHorizontalSNudge,
-    justifyContent: 'center',
     ...shorthands.padding(tokens.spacingVerticalM, tokens.spacingHorizontalMNudge),
   },
   mediumVertical: {
     // horizontal spacing is deliberate. This is the gap between icon and content.
     columnGap: tokens.spacingHorizontalSNudge,
-    justifyContent: 'flex-start',
-    minWidth: '120px',
```

Comment:
why was the `minWidth` removed?

Reply:
It was removed from the figma based on discussion with Yvonne.  It makes the icon only case too wide.